### PR TITLE
fix(icon): now getting from font awesome

### PR DESCRIPTION
**Pull request by daragh-king-genesys**
now getting from font awesome

✅ Closes: COMUI-2683

### CODE REVIEW

File: packages/genesys-spark-components/src/components/stable/gux-icon/icon-names-deprecated.ts [hunk 1]
```diff
@@ -171,27 +171,41 @@ export const deprecatedIconNamesMap: { [key: string]: string } = {
...
   'fa/eye-slash-regular': 'deprecated/fa/eye-slash-regular',
+  'fa/home-regular': 'fa/house-regular',
   'fa/life-ring-regular': 'deprecated/fa/life-ring-regular',
   'fa/link-horizontal-slash-regular': 'fa/link-slash-regular',
   'fa/link-horizontal-solid': 'fa/link-regular',
   'fa/list-radio-clean-regular': 'fa/list-ul-regular',
   'fa/list-radio-regular': 'fa/list-ul-regular',
   'fa/list-regular': 'fa/list-ul-regular',
   'fa/rectangle-list-regular': 'fa/square-list-regular',
+  'fa/regular-diagram-project-gear': 'custom/diagram-project-gear-regular',
```
Comment (321gillian) [inline packages/genesys-spark-components/src/components/stable/gux-icon/icon-names-deprecated.ts:194]: This and deprecated/fa/diagram-project-gear-regular are identical. Do we need both?

File: packages/genesys-spark-components/src/components/stable/gux-icon/icons/fa/xmark-large-regular.svg
Comment (321gillian) [inline packages/genesys-spark-components/src/components/stable/gux-icon/icons/fa/xmark-large-regular.svg:1]: While this is the correct svg, this change has the potential to cause breaking changes. 
Screenshot 2024-04-29 at 13 15 03
Screenshot 2024-04-29 at 13 09 59
-spark/assets/25255508/5b690167-dc40-4665-8157-334ea516ee50)

**Comment by 321gillian** [comment]
The carets in the content search seem a little out of alignment on this branch. 

This branch is shown on the left and the current WCL version is on the right:

Screenshot 2024-04-29 at 13 07 35

**Comment by daragh-king-genesys** [comment]
This will be a v5 issue

**Pull request closed without merging**